track/trackextractor.py

```diff
@@ -687,7 +687,7 @@ def generate_optical_flow(self):
     def start_and_end_in_secs(self, track):
         return (
             self.frame_time_in_secs(track, 0),
-            self.frame_time_in_secs(track, len(track) - 1),
+            self.frame_time_in_secs(track, len(track)),
```

Comment:
This freaks me out a bit because it is using a track index that doesn't exist.  But I can see that is what the rest of the code that reads the database might need.